src/plugins/explore/public/application/register_tabs.ts

```diff
@@ -32,6 +33,29 @@ export const registerBuiltInTabs = (tabRegistry: TabRegistryService) => {
 
   tabRegistry.registerTab(logsTabDefinition);
 
+  // Register Visualizations Tab
+  tabRegistry.registerTab({
+    id: 'explore_patterns_tab',
+    label: 'Patterns',
+    flavor: [],
+    order: 15,
+    supportedLanguages: ['PPL'],
+
+    prepareQuery: (query) => {
+      return query;
```

Comment:
this should add the patterns command?

Reply:
i thought we don't want to modify the visible query @ananzh?

Reply:
there's still some discussion about this, but for now i've just put it as a placeholder and i'm going to focus on this in another pr